srv/salt/_modules/osd.py

```diff
@@ -481,14 +481,23 @@ def quiescent(self):
         last = []
         if self.settings['delay'] == 0:
             raise ValueError("The delay cannot be 0")
+        message = "Unkown error"
         while i < self.settings['timeout']/self.settings['delay']:
             current = self.pg_states()
             if not current:
                 log.warning("PGs are not present")
-                return False
-            if len(current) == 1 and current[0]['name'] == 'active+clean':
-                log.warning("PGs are active+clean")
-                return True
+                return {"result": False, "message": "PGs are not present"}
+            if len(current) == 1:
+                if current[0]['name'] == 'active+clean':
+                    log.warning("PGs are active+clean")
+                    return {"result": True, "message": "PGs are active+clean"}
+                else:
+                    if 'scrub' in current[0]['name']:
+                        message = ("PGs are scrubbing, "
+                                   "disable scrubbing and retry.")
+                    else:
+                        message = "PGs state: " + current[0]['name']
+                    log.warning(message)
             log.warning("Waiting on active+clean {}".format(pprint.pformat(current)))
```

Comment:
This should possibly set `message` to the above string that's logged as a warning (otherwise this function may return "Unknown error")